src/worker/jobs/chainSubscribers.ts

```diff
@@ -218,10 +235,28 @@ class ChainSubscribers {
     async _processCommunityEvents(
         log: ethers.providers.Log
     ): Promise<ethers.utils.LogDescription | undefined> {
-        const parsedLog = this.ifaceCommunity.parseLog(log);
+        let parsedLog: ethers.utils.LogDescription;
+        if (log.address === config.communityContractAddress) {
+            parsedLog = this.ifaceNewCommunity.parseLog(log);
+        } else {
+            parsedLog = this.ifaceCommunity.parseLog(log);
+        }
```

Comment:
hmm, you can't do it this way because every community has a different address.
For the sake of simplicity, lets assume that, the new contracts do not have the manager address on the event, so past events have the same format. We just need to add new ones, all on the same ABI.